_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,700 @@
+
+            .layout-col-b {
+                @include span(6 of 12 last);
+            }
+        }
+    }
+}
+
+// 1/4 + 1/2 + 1/4 Layout
+.layout-1q1h1q {
+
+    // CASE: standard layout
+    &.layout {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-default) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(6 of 12);
+            }
+
+            .layout-col-c {
+                @include span(3 of 12 last);
+            }
+        }
+    }
+
+    // CASE: flush layout
+    &.layout-flush {
+        @extend %layout-flush;
+        @include susy-breakpoint($bp-screen-md, $susy-flush) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(6 of 12);
+            }
+
+            .layout-col-c {
+                @include span(3 of 12 last);
+            }
+        }
+    }
+
+    // CASE: reversed layout
+    &.layout-reversed {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-reversed) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(6 of 12);
+            }
+
+            .layout-col-c {
+                @include span(3 of 12 last);
+            }
+        }
+    }
+}
+
+// 1/4 + 1/4 + 1/2 Layout
+.layout-1q1q1h {
+
+    // CASE: standard layout
+    &.layout {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-default) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(3 of 12);
+            }
+
+            .layout-col-c {
+                @include span(6 of 12 last);
+            }
+        }
+    }
+
+    // CASE: flush layout
+    &.layout-flush {
+        @extend %layout-flush;
+        @include susy-breakpoint($bp-screen-md, $susy-flush) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(3 of 12);
+            }
+
+            .layout-col-c {
+                @include span(6 of 12 last);
+            }
+        }
+    }
+
+    // CASE: reversed layout
+    &.layout-reversed {
+        @extend %layout-reversed;
+        @include susy-breakpoint($bp-screen-md, $susy-reversed) {
+
+            .layout-col-a {
+                @include span(3 of 12 first);
+            }
+
+            .layout-col-b {
+                @include span(3 of 12);
+            }
+
+            .layout-col-c {
+                @include span(6 of 12 last);
+            }
+        }
+    }
+}
+
+// ----------------------------
+// #CANNED - VIEW
+// ----------------------------
+// 1/2 + 1/2 View Layout
+.layout-view-1h1h {
```

Comment:
NOTE: View-based layouts, these elements will span the entirety of the viewport. That's why they are prefixed - to help enforce that they are used for a different purpose.